examples/hello-string/wasmx

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env node
```

Comment:
What is this file? Did you write it?

Reply:
yes, in-lining the node code as in hello-world.sh would be a bit too much with all the imports

Reply:
I use it for emcc generated code as well, thus the demangle and strange imports

Reply:
makes no sense here, so I'll remove the clutter